small change (#2939)
wix-incubator/wix-code-docs: guides/api-query-language-velo.md

# Querying Velo APIs

The following query techniques are used in Velo. 

> Check the Velo Reference to see which technique is supported for querying a specific API. 

Reviewer: ```suggestion
Check a specific query function the Velo Reference to see which querying technique is supported. 
```

    
* [With the API Query Language](#without-query-builders): Some query functions retrieve a list of items using the API Query Language instead of with query builders. For these queries, pass an object defining the query to the query function.

## With query builders

For querying with query builders, call the query functions dedicated to a specific API that build the query. Each query function supports different sets of operators and paging options. For example, the Pricing Plans [`queryPublicPlans()`](https://www.wix.com/velo/reference/wix-pricing-plans-v2/plans/querypublicplans) function:
+ Processes the query specified with the [`PlansQueryBuilder`](https://www.wix.com/velo/reference/wix-pricing-plans-v2/plans/plansquerybuilder/ascending) operators.
+ Presents the results using the [`PlansQueryResult`](https://www.wix.com/velo/reference/wix-pricing-plans-v2/plans/plansqueryresult/currentpage) specifications. 

> Alternatively, you can call the more generic yet robust Wix Data API to query any collection. The Wix Data API offers additional query functionality like aggregation and querying referenced items. Call the [`query()`](https://www.wix.com/velo/reference/wix-data/query) function with: 

Reviewer: ```suggestion
Alternatively, you can call the more generic yet robust Wix Data API to query any collection in your site's CMS. The Wix Data API offers additional query functionality like aggregation and querying referenced items. Call the [`query()`](https://www.wix.com/velo/reference/wix-data/query) function with: 
```

+ [`DataItemsQueryBuilder`](https://www.wix.com/velo/reference/wix-data/wixdataquery) operators
+ [`DataItemsQueryResult`](https://www.wix.com/velo/reference/wix-data/wixdataqueryresult) specifications

## With the API Query Language

> For querying without query builders, pass an object defining the query to the query function. Define the object with the API Query Language.

Reviewer: ```suggestion
For querying without query builders, pass an object defining the query to the query function. Define the object with the API Query Language, shown in [The filter object](#the-filter-object) below.
```
